exercises/perfect-numbers/src/test/scala/PerfectNumbersTest.scala

```diff
@@ -0,0 +1,47 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class PerfectNumbersTest extends FlatSpec with Matchers {
```

Comment:
Does it make sense to add the tests to `x-common/exercises/perfect-numbers` first?